lib/ubi_cli.rb

```diff
@@ -394,7 +394,7 @@ def _submit_req(env)
   if Config.test? && ENV["CLOVER_FREEZE"] == "1"
     singleton_class.prepend(Module.new do
       def process(argv, env)
-        DB.block_queries do
+        DB.allow_queries do
```

Comment:
@jeremyevans , since the paths now need to pull locations from the database, `block_queries` here cause failure. I don't know what's the best way to solve this, can you please help me?

Reply:
Unfortunately no, I would like to dynamically be able to create new locations at the runtime. Can you please elaborate on 1#